File: tests/data-clumps/test-cases/parameter-field/negative-insufficient-shared/typescript/source/Patient.ts
export class Patient {
  public firstName: string;
  public lastName: string;
  public insuranceNumber: string;

  constructor(firstName: string, lastName: string, insuranceNumber: string) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.insuranceNumber = insuranceNumber;
  }
}

File: tests/data-clumps/test-cases/field-field/todo-different-types-same-type-name/positive/java/source/patient/Patient.java
package patient;

import doctor.Address; // Importing Address from doctor package

public class Patient {
  public String firstname;
  public String lastname;
  public doctor.Address address; // this is the same as Doctor's Address class, because it is imported from doctor package

    public Patient(String firstname, String lastname, doctor.Address address) {
        this.firstname = firstname;
        this.lastname = lastname;
        this.address = address;
    }
}